discovery_engine/lib/src/domain/engine/engine.dart

```diff
@@ -141,7 +141,7 @@ abstract class Engine {
   );
 
   /// Returns the currently trending topics.
-  Future<List<TrendingTopic>> getTrendingTopics();
+  Future<List<TrendingTopic>> trendingTopics();
```

Comment:
:grey_question: Isn't this a breaking change for blue?

Reply:
no, this is just the internal engine, the events stay the same.